Indicators/HilbertTransformIndicator.cs

```diff
@@ -0,0 +1,162 @@
+using System;
```

Comment:
Please add QuantConnect's Apache License headers to cs files.

Reply:
fixed by ffd25a4d0dcac7f84e52f2b616fd42ad4a437119